apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Footer/Footer.spec.tsx

```diff
@@ -45,116 +46,67 @@ describe('Footer', () => {
     jest.resetAllMocks()
   })
 
-  it('should display Footer attributes', () => {
-    const { getByText } = render(
-      <JourneyProvider
-        value={{
-          journey: {
-            id: 'journeyId',
-            chatButtons: [
-              {
-                id: '1',
-                link: 'https://m.me/user',
-                platform: ChatPlatform.facebook
-              },
-              {
-                id: '2',
-                link: 'viber://',
-                platform: ChatPlatform.viber
-              }
-            ]
-          } as unknown as Journey,
-          variant: 'admin'
-        }}
-      >
-        <Footer />
-      </JourneyProvider>
+  it('should display Footer attributes', async () => {
```

Comment:
There are no async actions in this test
```suggestion
  it('should display Footer attributes', () => {
```